### ISSUE TITLE
better getCenter method for polygons

### ISSUE THREAD

**Issue by farfromrefug** [CartoDB/mobile-sdk#401]
Right now the default implementation for `getCenter` of a polygon is not that good. 
There is a better implemetation in mabpox https://github.com/mapbox/polylabel.
Would be nice to use it in carto SDK.

**Comment by mtehver** [comment]
Correct, the current implementation is very simplistic and gives good results only for convex polygons/polygons not containing any holes. There is a better implementation in internal GeomUtils module, but this is not exposed via API currently. The 'polylabel' approach seems even better, but seems more expensive to calculate. I am currently marking this with 'idea-maybe-future'.
